plugins/package-managers/node/src/main/kotlin/yarn2/Yarn2.kt

```diff
@@ -642,6 +642,8 @@ private data class PackageHeader(
 
 private val PackageHeader.moduleId: String get() = "$rawName@${version.cleanVersionString()}"
 
+private val PackageHeader.isProject: Boolean get() = type == "workspace"
```

Comment:
Nit: Maybe put before `moduleId` to maintain alphabetical sorting of extension functions?